cmd/build/info.go

```diff
@@ -0,0 +1,29 @@
+package build
+
+import "time"
+
+var (
+	version  = "dev"
```

Comment:
does it make sense to use a fixed value here

Reply:
you mean we should just leave it blank like the commit? fine by me